x-pack/plugins/infra/public/components/asset_details/links/link_to_alerts_page.tsx

```diff
@@ -33,7 +33,7 @@ export const LinkToAlertsPage = ({ assetName, queryField, dateRange }: LinkToAle
   );
 
   return (
-    <RedirectAppLinks coreStart={services}>
+    <RedirectAppLinks coreStart={services} style={{ display: 'inline-block' }}>
```

Comment:
Q: Why do we need this change?

Reply:
Good question. Because of the line break before the `alerts` link

<img width="717" alt="image" src="https://github.com/elastic/kibana/assets/2767137/3cd41592-a98f-474d-bb5f-877d1d4f7f98">

Reply:
just realized this change didn't affect it. Will look into it.

Reply:
Done. It should look like this now:
<img width="657" alt="image" src="https://github.com/elastic/kibana/assets/2767137/4e4da0a5-033a-4ff8-84ef-c1c44eb9332c">

Reply:
You're right. I'll remove this one.